.coveragerc

```diff
@@ -7,6 +7,7 @@ omit = matplotlib/_version.py
 
 [report]
 exclude_lines =
+    pragma: no cover
```

Comment:
is this intentional?

Reply:
Yes, see https://github.com/matplotlib/matplotlib/pull/8987#issuecomment-691373397